source-controller/install-source-controller.md

```diff
@@ -46,27 +47,39 @@ To install Source Controller:
     For example:
 
     ```console
-    $ tanzu package available get controller.source.apps.tanzu.vmware.com/0.3.3 --values-schema --namespace tap-install
-     Retrieving package details for controller.source.apps.tanzu.vmware.com/0.3.3...
-     KEY           DEFAULT  TYPE    DESCRIPTION
-     ca_cert_data           string  Optional: PEM Encoded certificate data for image registries with private CA.
+    tanzu package available get controller.source.apps.tanzu.vmware.com/0.4.1 --values-schema --namespace tap-install
+    
+    Retrieving package details for controller.source.apps.tanzu.vmware.com/0.4.1... 
+    KEY               DEFAULT  TYPE    DESCRIPTION                                                                        
+    aws_iam_role_arn           string  Optional: The AWS IAM Role ARN to attach to the source controller service account  
+    ca_cert_data               string  Optional: PEM Encoded certificate data for image registries with private CA.
     ```
 
-3. (Optional) Enable Source Controller to connect to image registries that use self-signed or private certificate authorities.
-If a certificate error `x509: certificate signed by unknown authority` occurs, this option can be used to trust additional certificate authorities.
+3. (Optional) There are two optional fields that can override Source Controller's default installation setting.
 
-    To provide a custom certificate, create a file named `source-controller-values.yaml` that includes the PEM-encoded CA certificate data.
+- `ca_cert_data` Enables Source Controller to connect to image registries that use self-signed or private certificate authorities.If a certificate error `x509: certificate signed by unknown authority` occurs, this option can be used to trust additional certificate authorities.
```

Comment:
Need to add a space character in this section: 
`private certificate authorities.If`
-- SHOULD BE --
`private certificate authorities. If`

Reply:
Done